libbeat/processors/add_cloud_metadata/provider_aws_ec2_test.go

```diff
@@ -198,6 +198,12 @@ func TestRetrieveAWSMetadataEC2(t *testing.T) {
 				"cloud": mapstr.M{
 					"instance": mapstr.M{"id": instanceIDDoc2},
 				},
+				"orchestrator": mapstr.M{
```

Comment:
I am not sure about this test here

based on [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/add-cloud-metadata.html):
```
The third optional configuration setting is `overwrite`. When overwrite is true, add_cloud_metadata overwrites existing `cloud.*` fields (false by default).
```

now when the `orchestrator` is not part of the `cloud` object, this will not be overwritten.
Any thought?

Reply:
Sorry for delaying to test this Tania. I think that this page strictly speaking does not refer to orchestrator at all.

I think we can leave it for now or of you want to add a note in the doc